app/views/errors/show.html.erb

```diff
@@ -0,0 +1,84 @@
+<html>
```

Comment:
the main layout has calls to the database thus it will create an infinite loop if the error is the database connection.